e2e/utils/TextUtil.ts

```diff
@@ -0,0 +1,22 @@
+import { ObjectUtil } from './ObjectUtil';
+import type { IText, Point } from 'fabric';
+
+export class TextUtil<T extends IText = IText> extends ObjectUtil<T> {
+  async getCanvasCursorPositionAt(index: number): Promise<Point> {
+    return this.executeInBrowser(
+      (object, { index }) => {
+        const { left, top, topOffset, leftOffset } =
+          object._getCursorBoundaries(index, true);
+        // add extra 10 pixels to avoid controls
+        const point = new fabric.Point(
```

Comment:
@ShaMan123 here i needed to use it.
I can't import Point here nor pass it as a context in executeInBrowser

Reply:
I see, thanks